ui/lib/core/addon/components/object-list-input.hbs

```diff
@@ -6,10 +6,10 @@
 {{yield}}
 {{#each this.inputList as |row index|}}
   {{#let (get @validationErrors index) as |rowValidations|}}
-    <div class="field is-grouped">
```

Comment:
class changes fix inline error expanding the third column
<img width="1109" alt="Screenshot 2023-11-14 at 5 24 21 PM" src="https://github.com/hashicorp/vault/assets/68122737/08ea0529-98c5-4747-8aad-a7858da52d4c">